tests/test_helpers.py

```diff
@@ -376,6 +377,10 @@ def send_order(client, ddo, datatoken, service, cons_wallet, expect_failure=Fals
         cons_wallet,
     )
     datatoken.verify_order_tx(
-        web3, tx_id, ddo.asset_id, service.index, amount, cons_wallet.address
+        tx_id, ddo.asset_id, service.index, amount, cons_wallet.address
     )
     return tx_id
+
+
+def get_web3():
```

Comment:
This method is effectively a duplicate of `ocean_provider.utils.basics.get_web3()` - just use the latter instead.